Source/Operators.swift

```diff
@@ -2,3 +2,4 @@ infix operator <^> { associativity left precedence 130 }
 infix operator <*> { associativity left precedence 130 }
 infix operator >>- { associativity left precedence 100 }
 infix operator -<< { associativity right precedence 100 }
+infix operator <<- { associativity right precedence 100 }
```

Comment:
If we were to add this in, it would be nice to have both `<<-` and `->>` included, like we do for `flatMap`.